### DISCUSSION TITLE
My GitHub account has been suspended, but I haven't heard from GitHub.

### DISCUSSION THREAD

**Discussion by droptheai** [community/community#127971] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

hello. My old account was suddenly suspended. It's been two weeks already and GitHub hasn't contacted me at all, just gave me an email and a ticket number: I'm having trouble finding a job because my account is blocked during the company application season. Please resolve this quickly. What can I do?
My GitHub account is email: letgoofthepizza@gmail.com username: pizzalist 
Ticket ID: 2811709

## Please do not write below this line ##

NOAH,

Thank you for contacting GitHub Support. We wanted to let you know that we've received your message. We are experiencing high volumes and therefore, you may experience longer than normal wait times. In the meantime, you may find answers to commonly asked questions in our [community forum](https://github.community/) or in our [documentation](https://docs.github.com/).

Ticket ID: 2811709

If you have any additional information or would like to add anything to your initial message, now would be a great time to do so, feel free to reply to this email. If not, then rest assured your request is in the right hands :)

Thank you!
The GitHub Support Team

This email is a service from GitHub Support.
[KLMW69-WLNGV]

**Comment by ebndev** [comment]
Hi @droptheai,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
